packages/fleather/lib/src/widgets/editor_toolbar.dart

```diff
@@ -1228,12 +1201,18 @@ class FLIconButton extends StatelessWidget {
 }
 
 class SelectorScope extends StatefulWidget {
+  final Widget child;
+
   const SelectorScope({super.key, required this.child});
 
   static SelectorScopeState of(BuildContext context) =>
       context.findAncestorStateOfType<SelectorScopeState>()!;
 
-  final Widget child;
+  static Future<T?> pushSelector<T>(
+          BuildContext context, Widget selector, Completer<T?> completer,
+          {bool rootOverlay = false}) =>
+      SelectorScope.of(context)
+          .pushSelector(context, selector, completer, rootOverlay: rootOverlay);
```

Comment:
We don't seem to use this option. We should remove it

Reply:
It's a public API and might be used by users implementing their own custom buttons. @amantoux

Reply:
Ok, but still I think we shouldn't offer the option if not required by a user. It adds complexity for no assessed benefit.
You can keep it if you want though, I don't really mind